### DISCUSSION TITLE
Portable for Linux

### DISCUSSION THREAD

**Discussion by ivanooi** [ImageMagick/ImageMagick#2476] [General]
Linux portable reported access denied when try to execute in the terminal. Any ideas ? Im using Ubuntu 20.

Thanks

**Comment by urban-warrior** [comment]
It would be helpful if you post the exact exception you received.  One possibility is ImageMagick portable must interact with FUSE.  There may be a permission problem with FUSE.

**Reply by ivanooi** [reply]
sorry, my fault... it should not run in a network drive. I must copy it to local drive and make it executable.
